tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-   py27, py3{5,6,7,8}
-   py27-functional, py3{5,6,7,8}-functional
+   py27, py3{5,6,7,8,9}
+   py27-functional, py3{5,6,7,8,9}-functional
```

Comment:
could you also update the tests in https://github.com/kubernetes-client/python-base?

Reply:
@roycaihw https://github.com/kubernetes-client/python-base/pull/217

Reply:
Thanks